integrated_channels/blackboard/client.py

```diff
@@ -630,7 +647,8 @@ def _get_bb_user_id_from_enrollments(self, user_email, course_id):
             HTTPStatus.NOT_FOUND.value
         )
 
-    def _get_or_create_integrated_grade_column(self, bb_course_id, grade_column_name, external_id, points_possible=100):
+    def _get_or_create_integrated_grade_column(self, bb_course_id, grade_column_name, external_id, points_possible=100,
```

Comment:
Anyone feel free to correct me if I'm wrong but I think our formatting convention for multiline params is
```
def method(
   self,
   arg1,
   arg2,
   arg3
)
```

Reply:
yeah, i'm far from an expert on this. I was working off this stackoverflow https://stackoverflow.com/a/24988191